### DISCUSSION TITLE
Beta 15 won't load

### DISCUSSION THREAD

**Discussion by 100below** [apohl79/audiogridder#1065] [Help]
Attempting to update to beta 15 from 1.1.1 M1. Startup screen crashes at "Loading plugins cache..."
Can I delete the cache manually, or try something else?
12.3.1
MacBook Pro (14-inch, 2021)
M1
<img width="640" alt="Screen Shot 2022-10-18 at 12 41 46 PM" src="https://user-images.githubusercontent.com/64507046/196498814-b2152324-0935-4c18-a422-977c83aa86ab.png">

**Comment by apohl79** [comment]
Here is how you can reset the plugins cache: https://github.com/apohl79/audiogridder/discussions/1024#discussioncomment-3635291 

But before removing the files, could I ask you to send them to me? I saw people reporting issues loading the cache before, but I've never been able to reproduce any of those errors. Cheers.

**Comment by 100below** [comment]
Hi Andreas,


Thanks for getting back and thanks for this excellent and important plugin.

Unfortunately I have a strict personal policy that I will only make changes
to my system in occasional breaks between sessions. I just installed the
latest Pro Tools, did a System upgrade and many plugin updates this week
and I am now back to work. So I will send you files and try the update
again when I take my next break. For now I am grateful to be backed up.


Once I am working again with the new AG, I will send along a new donation.
AG has been very helpful, and your continuing work is great.



Gene Lennon


https://www.facebook.com/GeneLennonProductions/
